Documentation/Signoffs/SpeedometerSystem.md

```diff
@@ -1,38 +1,34 @@
 SPEEDOMETER SUBSYSTEM
 --------------------
 ### Functions of the Subsystem
-The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on 4-digit a 7-segment display.
+The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on a LCD screen.
 ### Specifications & Constraints
--	Microcontroller must be compatible with MATLAB
--	Must be able to display the speed at which the user is going in the simulator
--	Shall connect to the PC running the simulation
--	Shall display the speed in real-time
--	Shall be powered by the PC	
-### Schematic
-##### 1. Pinouts
-
-![image](https://user-images.githubusercontent.com/117474540/202864081-4325652c-c3de-4a34-bf38-b649f1cd59cb.png)
+-	Microcontroller must be able to read data from MATLAB
+-	Shall connect to the PC running the simulation to obtain simulator data
+-	Shall display the speed at a refresh rate no slower than 100 ms	
 
-_Figure 1. Pinout of Arduino Uno_
+The refresh rate of less than 100 ms was chosen based off the acceptable latency of modern video games being between 20 ms and 100 ms [1]. Since human reaction time is around 200 ms, this refresh rate would be undetectable by the user of the simulation.
+### Schematic
+##### 1. Wiring Schematic
 
-![image](https://user-images.githubusercontent.com/117474540/202864050-6e376c04-8fbd-40f2-bc74-d78c4ec03fb3.png)
+![image](https://user-images.githubusercontent.com/117474540/205458447-3d4bf94d-3a07-4027-b8cf-cc1fce28f2ab.png)
 
-_Figure 2. Pinout of TM1637 4-digit 7-segment display_
+_Figure 1. Wiring schematic for the Arduino Uno and the 3.5 Inch TFT Color LCD Screen 
 
-##### 2. Wiring Schematic
+### Analysis
+-The 3.5 Inch TFT Color LCD Screen is designed to fit on an Arduino Uno. The UNO has proper connectivity via GPIO to the point where the user snaps the LCD screen on top of the UNO.
```

Comment:
The priority isn't to have something that works with Arduino. We have looked at displays using rasberry pi as well. With this LCD screen we can have it mounted in the car and have it display a power up animation like most cars have. Any display not revolving around an MCU will just be a monitor with an HDMI cable. The current system already has that so we figured a programmable display will allow us to add better finishing touches therefore giving the customer a better working product.

Reply:
My previous comments stand. The emphasis is on convenience rather than the most appropriate display. Choose the right display based on your customer and the application, then figure out how to interface to it. 

Still no 3d schematics.

Reply:
I am confused on why this is not an appropriate display. The LCD can display the speed from the simulation and can be programmed to display units, animations, etc.  Would it be more appropriate if we made a box around it to make it look like a TV screen?  The LCD does everything we need it to do at a good size and price.

Reply:
I am confused on why this is not an appropriate display. The LCD can display the speed from the simulation and can be programmed to display units, animations, etc. Would it be more appropriate if we made a box around it to make it look like a TV screen? The LCD does everything we need it to do at a good size and price. If you have multiple LCD's that do the same thing, wouldn't it be better for the process of design to pick the one that is the easiest to interface with?